homeassistant/components/device_sun_light_trigger.py

```diff
@@ -35,10 +48,10 @@ def setup(hass, config):
     light = get_component('light')
     sun = get_component('sun')
 
-    disable_turn_off = 'disable_turn_off' in config[DOMAIN]
+    disable_turn_off = CONF_DISABLE_TURN_OFF in config[DOMAIN]
```

Comment:
With this `disable_turn_off` can be `True` or `False`. Rather add `default=False` to the Schema and read the value here